exercises/K-functions-parameters/exercise3.js

```diff
@@ -1,5 +1,9 @@
 // Write your function here
-
-var greeting = createGreeting("Daniel");
+function createGreeting (name){
+    let createGreeting = "Hello, my name is ";
+    return createGreeting + name;
+}
+let greeting = createGreeting("Daniel.");
 
 console.log(greeting);
+ 
```

Comment:
yeah, nice that you have added spacing and understood the whole concept of function